internal/daemon/worker/auth_rotation_test.go

```diff
@@ -42,7 +42,7 @@ func TestRotationTicking(t *testing.T) {
 	const rotationPeriod = 20 * time.Second
 
 	w := worker.NewTestWorker(t, &worker.TestWorkerOpts{
-		InitialControllers: c.ClusterAddrs(),
```

Comment:
There are a several tests failing because they reference InitialControllers; can they be updated to InitialUpstreams?